test/test-creator-creator.js

```diff
@@ -16,6 +16,7 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
+import * as commonUtil from './common-util';
```

Comment:
Doesn't make much difference performance-wise but it's a good practice to import only what you need so I'd refrain from importing the entire `common-util` module namespace since you're only using a single method. Use a named import statement instead.